eventstore/models.py

```diff
@@ -197,6 +197,22 @@ class PrebirthRegistration(models.Model):
     data = JSONField(default=dict, blank=True, null=True)
 
 
+class PMTCTRegistration(models.Model):
+    NORMAL = "Normal"
+    HIGH = "High"
```

Comment:
I think these machine values should be `normal` and `high`, since that's what we use everywhere. We can keep the human values as `Normal` and `High`